app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -142,7 +142,7 @@ def self.storage_inv_to_hashes(inv)
 
           new_result = {
             :ems_ref            => mor,
-            :ems_ref_obj        => mor,
+            :ems_ref_type       => mor.vimType,
```

Comment:
I think vimType is a symbol here...is that ok?

Reply:
Yeah, the VimType setter does to_sym anyway, https://github.com/ManageIQ/vmware_web_service/blob/master/lib/VMwareWebService/VimTypes.rb#L11